components/org.wso2.carbon.identity.sso.saml.ui/src/main/resources/web/sso-saml/add_service_provider.jsp

```diff
@@ -826,7 +826,7 @@
                                         <font color="red">*</font>
                                     </td>
                                     <td><input type="text" id="issuer" name="issuer" maxlength="100"
-                                               class="text-box-big"
```

Comment:
What if we change this property in text-box-big class itself? will it break something?

Reply:
the class text-box-big is a global css property so, if we change the property there then the places where this property is used other than here will also change.